src/App.css

```diff
@@ -675,6 +625,10 @@ a.active {
   fill: var(--primary-color);
 }
 
+a.active .icon {
```

Comment:
This line is fixing the hover issue 😄 !

Reply:
Nice! I was suspecting a combo of some of those terms, but it likely would have taken me much longer than it took you to accomplish this. Thank you so much!